bgpd/bgp_evpn.c

```diff
@@ -85,10 +85,6 @@ static int vni_hash_cmp(const void *p1, const void *p2)
 	const struct bgpevpn *vpn1 = p1;
 	const struct bgpevpn *vpn2 = p2;
 
-	if (!vpn1 && !vpn2)
-		return 1;
-	if (!vpn1 || !vpn2)
-		return 0;
 	return (vpn1->vni == vpn2->vni);
```

Comment:
Why remove protection against null values?

Reply:
Because it's impossible for the calling code to pass in NULL's.  If either pointer is NULL we don't get to this function.